Classes/CRON/FormBuilder/Controller/FormBuilderController.php

```diff
@@ -230,13 +230,14 @@ protected function sendMail($fields, $files)
             $customerFields = [];
 
             foreach ($fields as $field) {
-
-                if ($field['node']->getProperty('type') == "email" && $field['node']->getProperty('isCustomerMail')) {
-                    $customerMail = $field['value'];
-                }
-
-                if ($field['node']->getProperty('type') == "name") {
-                    $customerName = $field['value'];
+                if (!empty($field)) {
```

Comment:
i prefer the "guard-condition" style:

```
            foreach ($fields as $field) {
                if (empty($field)) {
                    continue;
                }

                ...
            }
```

as it makes the body of the loop less indented and easier to follow. But its a matter of taste.

Reply:
I also find more beautiful I will remember.